apps/server/src/Scoring.ts

```diff
@@ -0,0 +1,60 @@
+let correctSequence = ['A', 'A', 'B', 'C', 'D'];
```

Comment:
If we are not gonna use this as a default sequence, should we make this an empty arr?